.github/workflows/main.yaml

```diff
@@ -0,0 +1,75 @@
+name: basic tests
+
+on:
+  push:
+    branches:
+      - 'user/**'
+      - 'feature/**'
+      - 'improvement/**'
+      - 'bugfix/**'
+      - 'w/**'
+      - 'q/**'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildk
+        uses: docker/setup-buildx-action@v1
+        with:
+          buildkitd-flags: --debug
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          context: eve/worker/dockerfile
+          file: eve/worker/dockerfile/actions.Dockerfile
+          build-args: BUILDBOT_VERSION=2.0.1
+          tags: "ghcr.io/scality/bert-e:${{ github.sha }}"
+
+  tests:
+    runs-on: ubuntu-latest
+    container:
+      image: "ghcr.io/scality/bert-e:${{ github.sha }}"
+      options: --user=0
+    needs:
+      - publish
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: tox -e utests
+      - run: tox -e tests-api-mock
+      - run: tox -e tests-noqueue
+      - run: tox -e tests
+      - run: tox -e coverage-report
+      - run: tox -e tests-server
```

Comment:
Can we split all those into a specific a job? Would it make sense?
Per example:
```yaml
  utests:
    runs-on: ubuntu-latest
    container:
      image: "ghcr.io/scality/bert-e:${{ github.sha }}"
      options: --user=0
    needs:
      - publish
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - run: tox -e utests
  tests-api-mock:
    runs-on: ubuntu-latest
    container:
      image: "ghcr.io/scality/bert-e:${{ github.sha }}"
      options: --user=0
    needs:
      - publish
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - run: tox -e tests-api-mock
... # and so on
```

Reply:
Well, we can compare this as eve. If this make sens in eve, it should in actions. But i am not really shure about it. 

 But put one or two that take long time,  in parallel could save a lot of time

Reply:
it can be done in eve (I don't know why it's not done yet)

I would say to split at least `tox -e tests` and `tox -e tests-noqueue` the rest can live together